lib/src/deprecation.dart

```diff
@@ -15,7 +15,7 @@ enum Deprecation {
   // DO NOT EDIT. This section was generated from the language repo.
   // See tool/grind/generate_deprecations.dart for details.
   //
-  // Checksum: 47c97f7824eb25d7f1e64e3230938b88330d40b4
+  // Checksum: b715850a96d4d346e09fecb54444ccec6dc69610
```

Comment:
Not sure what's going on here. The checksum is correct for the YAML file the marks `bogus-combinators` as obsolete, but the actual code is only updated `mozDocument`, not `bogusCombinators`

Reply:
Oh wait, now I figured it out. The `obsolete` value was added to the YAML, but the status wasn't changed from `active`